src/Symfony/Component/Validator/Constraints/NaivePasswordStrengthEstimator.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+use Symfony\Component\Validator\Exception\UnexpectedValueException;
+use ZxcvbnPhp\Matchers\DictionaryMatch;
+use ZxcvbnPhp\Matchers\MatchInterface;
+
+final class NaivePasswordStrengthEstimator
+{
+    /**
+     * @param string[] $restrictedData
+     *
+     * @return array{score: int, restrictedData: string[]}
+     */
+    public function __invoke(#[\SensitiveParameter] mixed $password, array $restrictedData): array
```

Comment:
Instead of having this empty callable by default maybe we can implement something simple like the one spot here: https://github.com/symfony/symfony/pull/49831#issuecomment-1485280176
What do you think?